### PR TITLE
Implement the new timeline.

### PR DESCRIPTION
**Pull request by msramek**
Day:
  0 - first notification
 13 - second notification
 14 - second notification again, open the article, and uninstall

Changed the accelerated timeline, so that the interval spacing is not always one minute.

### CODE REVIEW

File: chrome/test/sunset-test.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 test(function () {
   assert_equals(Sunset.timeline_offsets_.length, 3);
   assert_equals(Sunset.timeline_offsets_[0], 0);
-  assert_equals(Sunset.timeline_offsets_[1], 14);
-  assert_equals(Sunset.timeline_offsets_[2], 15);
+  assert_equals(Sunset.timeline_offsets_[1], 13);
+  assert_equals(Sunset.timeline_offsets_[2], 14);
```
Comment (mikewest) [inline chrome/test/sunset-test.js]: Can you also expand the shouldShowNotification_ test below?

File: chrome/sunset.js [hunk 2]
```diff
@@ -158,6 +157,32 @@ Sunset.showNotification_ = function(index) {
...
+ * @private
+ */
+Sunset.onAlarm_ = function(alarm) {
+  // Consider showing a notification on the regular clock alarm.
+  if (alarm.name == "clock")
+    Sunset.maybeShowNotification_();
+
+  // Uninstall the extension and show an article on the uninstall delay alarm.
+  if (alarm.name == "uninstall") {
+    Sunset.showArticle_();
```
Comment (mikewest) [inline chrome/sunset.js]: Leave this out for the moment, as I think we haven't reached a decision about whether or not it's the right thing to do.

File: chrome/sunset.js [hunk 3]
```diff
@@ -158,6 +157,32 @@ Sunset.showNotification_ = function(index) {
   chrome.storage.sync.set({
     "index": index + 1
   });
+
+  // Uninstall the extension a minute after the last notification.
+  if (index >= Sunset.timeline_offsets_.length - 1) {
+    chrome.alarms.create("uninstall", {
+        "delayInMinutes": 1
```
Comment (mikewest) [inline chrome/sunset.js]: How about 5 minutes?

**Comment by mikewest** [comment]
LGTM % comments. Land it when you've addressed them. :)

File: chrome/sunset.js (hunk 2 above)
Comment (mikewest) [inline chrome/sunset.js]: Nevermind, I should read my email. :)

File: chrome/test/sunset-test.js (hunk 1 above)
Comment (msramek) [inline chrome/test/sunset-test.js]: As discussed offline, there's nothing to update in the tests now.

File: chrome/sunset.js (hunk 3 above)
Comment (msramek) [inline chrome/sunset.js]: Done.

**Pull request merged**